conf/broker.conf

```diff
@@ -35,6 +35,11 @@ brokerServicePort=6650
 # Broker data port for TLS - By default TLS is disabled
 brokerServicePortTls=
 
+# Specify the ssl provider for the broker service:
+# When using the CACert, available values are one of OpenSSL and JDK.
+# When using the KeyStore, available values like SunJSSE and Conscrypt.
```

Comment:
```suggestion
# When using TLS authentication with KeyStore, available values can be SunJSSE, Conscrypt and etc.
```

Reply:
Thanks @momo-jun, should be resolved now.